pyproject.toml

```diff
@@ -20,5 +20,11 @@ pre-commit = "2.20.*"
 [tool.poetry.group.test.dependencies]
 pytest = "7.1.*"
 
+[tool.poetry.group.frontend.dependencies]
+fastapi = "0.95.*"
+jinja2 = "3.1.*"
+python-multipart="0.0.*"
```

Comment:
Do czego to jest używane?

Reply:
Faktycznie już niepotrzebne, było w początkowej wersji webservera